package/rm2-suspend-fix/package

```diff
@@ -7,7 +7,7 @@ pkgdesc="Fix issue where suspend breaks networking on the reMarkable 2"
 url=https://toltec-dev.org/
 pkgver=0.0.0-1
```

Comment:
```suggestion
pkgver=0.0.0-2
```

Reply:
I really thought I had found them all when I was skimming through this morning haha.